snapcraft/tests/unit/plugins/test_jhbuild.py

```diff
@@ -18,7 +18,8 @@
 import snapcraft.plugins.jhbuild
 
 
-class JHBuildPluginTestCase(snapcraft.tests.unit.TestCase):
+# LP: #1733584
+class JHBuildPluginTestCase(snapcraft.tests.unit.TestCase):  # type: ignore
```

Comment:
How is this ignore (and all the other test case ones below) related to that bug?

Reply:
There are a bunch of issues in `mypy` related to using `__init__.py`, one of them is that for this to work you would need to `import snapcraft.tests.unit.__init__` or something like that since it sort of works at the module level and not really at a package level. This is somewhat not urgent as it is discouraged to implement things in `__init__.py` in the python world.

Reply:
Ah, so the issue here is the fact that `TestCase` is in `snapcraft.tests.unit.__init__.py`? Okay, that makes more sense.